### PR TITLE
72 variables consistency validation

### PR DESCRIPTION
**Pull request by nicdub**
Check consistency between data conventions and variable dictionary

### CODE REVIEW

File: Data/AGG_Data_Convention.md [hunk 1]
```diff
@@ -2,13 +2,11 @@
```
Comment (charlottegiseleweil) [inline Data/AGG_Data_Convention.md:2]:
```suggestion

AGG Data constitutes input of S-N Curve module, together with
* ASTM.csv, available from https://www.astm.org/stp313-eb.html pp 20-23
* Reliability level parameter (RELIABILITY_LEVEL P(N) with value [0,1], default value : 0.5)
```

Suggestion to document parameter, that is not exactly AGG data. Clearer no?

File: Data/AGG_Data_Convention.md [hunk 2]
```diff
@@ -18,8 +16,13 @@ Encoding format: UTF-8, Separator: ',' (comma)
...
+| stress_cluster_number | Index of the cluster of stress level for Whitney method |  | [-] | int   | y                  |
 | stress_max            | Max stress                             | sigma_max | [MPa]   | double    | y                  |
 | cycles_to_failure     | Number of cycles to failure            | N         | [-]     | int       | y                  |
 | residual_strength     | Residual strength                      | sigma_r   | [MPa]   | double    | y for Sendeckyj    |
+
+### AGG parameters
+
+| Variable name         | Description                            | Symbol    | Unit    | Data type | Default value     |
+|-----------------------|----------------------------------------|-----------|---------|-----------|-------------------|
+| RELIABILITY_LEVEL     | Reliability level                      | P(N)      | 0-1.0   | double    | 0.5               |
```
Comment (charlottegiseleweil) [inline Data/AGG_Data_Convention.md:28]:
```suggestion
```

since documented above

File: Data/CLD_Data_Convention.md [hunk 3]
```diff
@@ -4,8 +4,8 @@ Note: in all the Fortran codes, the output range of mean stress needs to be chec
```
Comment (charlottegiseleweil) [inline Data/CLD_Data_Convention.md:4]:
```suggestion
CLD Data is the output of the Constant-Life Diagram module.
```

File: Data/CYC_Data_Convention.md [hunk 4]
```diff
@@ -2,7 +2,7 @@
```
Comment (charlottegiseleweil) [inline Data/CYC_Data_Convention.md:2]:
```suggestion
CYC Data is the output of the Cycle Counting module.
```

File: Data/CYC_Data_Convention.md [hunk 5]
```diff
@@ -12,11 +12,10 @@ with:
...
-| stress_ratio         | Stress ratio                           | R           | [-]     | double    | y                  |
-| n_cycles?            | ?                                      | ?           |         | double    | y                  |
-| cum_n_cycles?        | percentage?                            | ?           |         | double    | y                  |
+| Variable name | Description                   | Symbol      | Unit  | Data type | Mandatory |
+| ------------- | ----------------------------- | ----------- | ----- | --------- | --------- |
+| stress_range  | Stress range (= 2 \* sigma_a) | delta_sigma | [MPa] | double    | y         |
+| stress_mean   | Mean stress                   | sigma_mean  | [MPa] | double    | y         |
+| stress_ratio  | Stress ratio                  | R           | [-]   | double    | y         |
+| n_cycles      | Number of cycles              | n           |       | double    | y         |
+| cum_n_cycles  | percentage                    |             |       | double    | y         |
```
Comment (charlottegiseleweil) [inline Data/CYC_Data_Convention.md:21]:
```suggestion
| cum_n_cycles  | cumulative number of cycles for a given bin of sigma_a, sigma_mean and R.                    |             |       | double    | y         |
```

File: Data/DAS_Data_Convention.md [hunk 6]
```diff
@@ -4,8 +4,8 @@ Note: Currently module 5 re-runs module 3 inside module 5. Needs to be updated.
```
Comment (charlottegiseleweil) [inline Data/DAS_Data_Convention.md:4]: Can you remove the Note "Currently module 5 re-runs module 3 inside module 5. Needs to be updated." this should be converted to an issue if you think it's worth noting (not sure it is), else you'll just refactor naturally when migrating the functions of modules 3 and 5 ! 

Also add description of this dataset : 

```suggestion
DAS Data corresponds to the output of Damage summation module.
```

File: Data/FAF_Data_Convention.md [hunk 7]
```diff
@@ -2,8 +2,8 @@
```
Comment (charlottegiseleweil) [inline Data/FAF_Data_Convention.md:2]:
```suggestion
FAF Data corresponds to the output of Fatigue failure module.
```

File: Data/SNC_Data_Convention.md [hunk 8]
```diff
@@ -1,19 +1,18 @@
 # SNC Standard Data format
 
 Each module 2 (S-N curve) produce two files:
```
Comment (charlottegiseleweil) [inline Data/SNC_Data_Convention.md:3]:
```suggestion
SNC Data corresponds to the output of S-N Curve module and is composed of 2 files :
```

File: Data/SNC_Data_Convention.md [hunk 9]
```diff
@@ -23,7 +22,7 @@ Encoding format: UTF-8
 [
 	{
 		"stress_ratio": double, // Stress ratio (R) [-]
-		"reliability_level": int, // Reliability level (rsql) [-]
+		"confidence_interval": double, // confidence bounds, usually 5, 95% (rsql) [-]
 		"a": double, // S-N Curve parameter [-]
 		"b": double, // S-N Curve parameter [1/MPa]
 		"lrsq": double, // [?]
```
Comment (charlottegiseleweil) [inline Data/SNC_Data_Convention.md:28]: aren't you missing fp here ??

**Review by charlottegiseleweil — APPROVED** [review]
just double check the fp thing in SNC.json - sinon ca m'a l'air tout bon si tu acceptes mes suggestions :)
(Don't forget to squash commits when accepting PR reviews , si tu ne sais pas faire on peut regarder ensemble avec Pierre ou Sam demain au meeting d'équipe?)
Merci !

File: Data/SNC_Data_Convention.md (hunk 9 above)
Comment (nicdub) [reply Data/SNC_Data_Convention.md:28]: you're right, fp is missing, I forget it because it's not used in all methods (not mendatory)

**Pull request merged**